pkg/api/types.go

```diff
@@ -2563,4 +2661,12 @@ type RangeAllocation struct {
 const (
 	// "default-scheduler" is the name of default scheduler.
 	DefaultSchedulerName = "default-scheduler"
+
+	// RequiredDuringScheduling affinity is not symmetric, but there is an implicit PreferredDuringScheduling affinity rule
+	// corresponding to every RequiredDuringScheduling affinity rule.
+	// DefaultHardPodAffinityWeight represents the default weight of implicit PreferredDuringScheduling affinity rule.
+	DefaultHardPodAffinitySymmetricWeight int = 1
+
+	//DefaultFailureDomains string = "kubernetes.io/hostname,failure-domain.beta.kubernetes.io/zone,ailure-domain.beta.kubernetes.io/region"
```

Comment:
This is not a very helpful comment. ;-) Maybe "When the --failure-domains scheduler flag is not specified, DefaultFailureDomains defines the set of label keys used when TopologyKey is empty in PreferredDuringScheduling anti-affinity."